DZ-8/src/main/java/org/example/StudentsGroup.java

```diff
@@ -0,0 +1,98 @@
+package org.example;
+import java.util.List;
+
+public class StudentsGroup {
+    // Дані про групу, private
+    private Student leader; // Староста
+    private List<Student> students; // Список студентів
+    private List<String> tasks; // Список завдань
+
+    public StudentsGroup(Student leader, List<Student> students, List<String> tasks) {
```

Comment:
Constructor must be `public StudentsGroup(Student leader)` everything else cold be defined inside